lib/tasks/trends.rake

```diff
@@ -1,19 +1,11 @@
 namespace :trends do
```

Comment:
I believe this whole namespace should be removed, it was moved to the plugin

Reply:
done